Add requisites and installations to README (#11)
giantswarm/fluent-logshipping-app: README.md

- A fluentbit daemonset: Small collector running on every node collecting containers and audit logs and for forwarding them to fluentd.
- A fluentd deployment: Takes care of persisting the logs to any data sink

## Requirements

> - You can install only one release of this chart per kubernetes cluster

Author: yeah now it is not possible but it is on the roadmap

> - By default any forward is active so make sure you check [configuration document](helm/fluent-logshipping-app/Configuration.md) before deploy it in your cluster.

Reviewer: ```suggestion
- By default any forward is active so make sure you check [configuration document](helm/fluent-logshipping-app/Configuration.md) before deploying it in your cluster.
```

> - By default any forwarded is active so make sure you check [configuration document](helm/fluent-logshipping-app/Configuration.md) before deploy it in your cluster.

Reviewer: Is `forward` a thing that can be active? or should it be `forwarded logs are active`? or `any log forwarding is active` 

And what do you mean with `any` ? Should it be `all` ?

```suggestion
- By default any forward is active so make sure you check [configuration document](helm/fluent-logshipping-app/Configuration.md) before deploy it in your cluster.
```

Reviewer 2: I think it should be "no forward" as I had to disable all of them by default when I added Azure support

## Installation

> The logging forwarding app is built to be installed in AWS or Azure. For AWS support to send the logs to S3 and/or Cloudwatch. For Azure it uses Log Analytics:

Reviewer: ```suggestion
The logging forwarding app is built to be installed in AWS or Azure. On AWS the app can send logs to S3 and/or Cloudwatch. On Azure it uses Log Analytics.

Sample command for installing it on AWS with cloudwatch enabled:
```
